lib/utils/docker-coffee.d.ts

```diff
@@ -25,8 +25,13 @@ export interface BuildDockerOptions {
 	docker?: string; // dockerode DockerOptions.socketPath
 	dockerHost?: string; // dockerode DockerOptions.host
 	dockerPort?: number; // dockerode DockerOptions.port
+	host?: string;
+	port?: number;
+	timeout?: number;
```

Comment:
These types needed to be added for the sake of lib/actions/local/common.js which uses them